src/diffusers/models/normalization.py

```diff
@@ -566,3 +570,69 @@ def __init__(self, p: int = 2, dim: int = -1, eps: float = 1e-12):
 
     def forward(self, hidden_states: torch.Tensor) -> torch.Tensor:
         return F.normalize(hidden_states, p=self.p, dim=self.dim, eps=self.eps)
+
+
+class LayerNormNd(nn.LayerNorm):
```

Comment:
similar to https://github.com/huggingface/diffusers/pull/10021/files#r1861677920
I think we don't need to inherit here

Reply:
Updated. This requires model weights to be converted again, so will upload shortly